CONTRIBUTING.md

```diff
@@ -0,0 +1,155 @@
+  - other issues may require C++ changes in either the Brave code or in Chromium
+
+## Getting started
+* Make sure you have a [GitHub account](https://github.com/join).
+* Submit a [ticket](https://github.com/brave/brave-browser/issues) for your issue if one does not already exist. Please include the Brave version, operating system, and steps to reproduce the issue.
+* Fork the repository on GitHub (this might be [`brave-browser`](https://github.com/brave/brave-browser), [`brave-core`](https://github.com/brave/brave-core), or both).
+* For changes to JavaScript files, we recommend installing a [Standard](http://standardjs.com/) plugin for your preferred text editor in order to ensure code style consistency.
+* For C++ changes, you can consider setting up [clang-format](https://chromium.googlesource.com/chromium/src/+/master/docs/sublime_ide.md#Format-Selection-with-Clang_Format-Chromium-only) for your editor.
+
+### Making changes
+Once you've cloned the repo to your computer, you're ready to start making edits!
+
+Please note that there are two repositories here:
+* the root project ([`brave-browser`](https://github.com/brave/brave-browser)), which pulls down all of the Chromium code into `src/`
+* [`brave-core`](https://github.com/brave/brave-core) which is located on disk under the root at `src/brave`
+
+Depending on which you're editing, you'll need to add your fork to the remotes list. By default, `origin` is set to upstream.
+For example, here's how GitHub user `bsclifton` would add BOTH remotes `brave-browser` and `brave-core`:
+```sh
+# root where project is cloned
+cd ~/brave-browser/
+git remote add bsclifton git@github.com:bsclifton/brave-browser.git
+git fetch bsclifton
+# root for the `brave-core` repo
+cd src/brave
+git remote add bsclifton git@github.com:bsclifton/brave-core.git
+git fetch bsclifton
+```
+
+Once you're setup, there's a few tips we can suggest:
+
+* Make a new branch for your work. It helps to have a descriptive name, like `fix-fullscreen-issue`.
+* Make commits in logical units. If needed, run `git rebase -i` to squash commits before opening a pull request.
+* New features and most other pull requests require a new [test](https://github.com/brave/brave-browser/wiki/Tests) to be written before the pull request will be accepted.  Some exceptions would be a tweak to an area of code that doesn't have tests yet, text changes, build config changes, things that can't be tested due to test suite limitations, etc.
+* Use GitHub [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message, and make the commit message body as descriptive as necessary. Ex:
+
+````
+    Add contributing guide
+
+    This is a first pass at a contributor's guide so now people will know how to
+    get pull requests accepted faster.
+
+    Fix #108
```

Comment:
Updated! (just put the whole link, since that's usually what I do and I've seen others too. GitHub will reduce it down to the `brave/brave-browser#108` format)